src/beamlime/applications/_random_data_providers.py

```diff
@@ -24,108 +18,37 @@
 RandomEvents = NewType("RandomEvents", List[sc.DataArray])
 
 
-def provide_rng(random_seed: RandomSeed) -> RNG:
-    from numpy.random import default_rng
-
-    return default_rng(random_seed)
-
+DetectorNumberCands = NewType("DetectorNumberCands", List[int])
```

Comment:
What is `Cands`?

Reply:
``Candidates`` sorry...
Since the geometry will carry all possible ``pixel id``s, not a range,
it can just receive that list and choose among them.

Reply:
Can you rename it?